DGN/Models/Article.cs

```diff
@@ -23,36 +23,31 @@ public class Article
         [DataType(DataType.MultilineText)]
         public string Body { get; set; }
 
-        [DisplayName("The article image location")]
+        [DisplayName("Image Location")]
         public string ImageLocation { get; set; }
 
         [Required]
-        [ForeignKey("Category")]
+        [DisplayName("Category")]
         public int CategoryId { get; set; }
 
-        [Required]
         public Category Category { get; set; }
 
-        [ForeignKey("Author")]
-        public int? AuthorId { get; set; }
+        public int? UserId { get; set; }
 
-        public User Author { get; set; }
+        public User User { get; set; }
 
-        [Required]
         [DisplayName("Creation Timestamp")]
         [DataType(DataType.Date)]
         [DisplayFormat(DataFormatString = "{0:dd-MM-yyyy}", ApplyFormatInEditMode = true)]
         public DateTime CreationTimestamp { get; set; } = DateTime.Now;
 
-        [Required]
         [DisplayName("Last Updated Timestamp")]
         [DataType(DataType.Date)]
         [DisplayFormat(DataFormatString = "{0:dd-MM-yyyy}", ApplyFormatInEditMode = true)]
         public DateTime LastUpdatedTimestamp { get; set; } = DateTime.Now;
 
         public IList<Comment> Comments { get; set; } = new List<Comment>();
 
-        [InverseProperty("ArticleLikes")]
-        public IList<User> Likes { get; set; } = new List<User>();
+        public IList<User> UserLikes { get; set; } = new List<User>();
```

Comment:
יש לבדוק האם זאת הדרך הנכונה לאתחל את המתשנה הזה.

Reply:
על איזה משתנה את מדברת, יש פה פשוט כמה?
אם זה הלייקים אז כן רשימה ריקה של יוזרים שעשו לייק למה לא?

Reply:
I checked and It's ok to initialize those fields in that way.
But we need to remember that if we want the controller to create a model **with** the DEFAULT value we need to remove the variable from the form and bind

for example if we want the default value for `Role` when creating a user so we should change:

Create([Bind("Id,Email,Username,Firstname,Lastname,Birthday,**Role**,ImageLocation,About")] User user)

to:
Create([Bind("Id,Email,Username,Firstname,Lastname,Birthday,ImageLocation,About")] User user)